covfee/server/orm/task.py

```diff
@@ -134,9 +134,27 @@ def pause(self, pause: bool):
         self.get_task_object().on_admin_pause()
 
     def make_results_dict(self):
-        return {
-            "responses": [response.make_results_dict() for response in self.responses]
-        }
+        results_list = []
+        for response in self.responses:
+            result_dict = response.make_results_dict()
+
+            result_dict["annotations"] = [
+                utils.NoIndentJSON(annotation.data_json)
+                for annotation in self.annotations
+                if annotation.data_json is not None
+            ]
+
+            if (
+                self.journey_associations is not None
```

Comment:
Rather than using `journey_associations` couldn't you use `journeys` https://github.com/TUDelft-SPC-Lab/covfee/blob/a433d5295759f0ab891b0c1ed0e92cd1267711d9/covfee/server/orm/node.py#L158 AFAIK It's supposed to be populated thanks to the indirect relationship established by `journey_associations`

Reply:
Fixed in f0168e267a72fb12134e1ae3ea09b0c6d0db22c5